addons/stock/static/src/js/inventory_validate_button_controller.js

```diff
@@ -71,6 +81,23 @@ var InventoryValidationController = ListController.extend({
                     self.do_notify(
                         _t("Success"),
                         _t("The inventory has been validated"));
+
+                    // The presence of 'button_validate_picking_ids' in the context means the call
+                    // was made from a Zero Quantity Count, wich means there's no need to redirect
+                    // to that inventory.
+                    if (!(self.context.button_validate_picking_ids)) {
```

Comment:
Traceback when validate an inventory from the classical form view (seems `self.context` is undefined).

Reply:
This appears to no longer be the case. Maybe you should re-confirm during the next review?